lib/Ohmbrewer_Menu/Ohmbrewer_Menu_WiFi.cpp

```diff
@@ -12,7 +12,7 @@
     //Set up list of options in vector
     options.push_back("WiFi On");
     options.push_back("WiFi Off");
-    _selectedOption = _settings->isWifiOn();
+    _selectedOption = 0;
```

Comment:
What if when the wifi what shut off when the Rhizome was last running? We want to load the settings from EEPROM, right? The constructor for _settings will load the setting, but we need to pass it into the menu to get the view to show what's actually selected.

This would only really be an issue on the very first time the user goes to this menu. The change would make it so in this situation the wifi would actually be off, but the state of the menu wouldn't change until the user selects an option.